asterius/rts/rts.memory.mjs

```diff
@@ -56,6 +56,16 @@ export class Memory {
   i64LoadU16(p) { return BigInt(this.dataView.getUint16(Memory.unTag(p), true)); }
   heapAlloced(p) { return Memory.unTag(p) >= (this.staticMBlocks << Math.log2(rtsConstants.mblock_size)); }
   strlen(_str) { return this.i8View.subarray(Memory.unTag(_str)).indexOf(0); }
+  strLoad(_str) {
+      let p = Memory.unTag(_str);
+      let s = "";
+
+      for(let i = 0; i < this.strlen(_str); ++i) {
```

Comment:
This loop traverses the c string twice, first finding its length and then converting to js string. We don't need two traversals.

Reply:
Reverting back to old `while(1) {}` style